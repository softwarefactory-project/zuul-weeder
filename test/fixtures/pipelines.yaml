- pipeline:
    name: check
    trigger:
      my_gerrit: []
      my_gitlab: []
    success:
      elasticsearch:
    failure:
      my_gerrit:
