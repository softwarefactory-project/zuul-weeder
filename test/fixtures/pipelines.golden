[ ZPipeline
    ( Pipeline
        { name = PipelineName "check"
        , triggers =
            [ PipelineTrigger
                { connectionName = ConnectionName "my_gerrit" }
            , PipelineTrigger
                { connectionName = ConnectionName "my_gitlab" }
            ]
        }
    )
]